src/utils.h

```diff
@@ -28,7 +28,12 @@ void convertUint256BE(uint8_t* data, uint32_t length, uint256_t* target);
 
 int local_strchr(char* string, char ch);
 
-uint32_t getV(txContent_t* txContent);
+// `itoa` for uint32_t.
+void u32_to_str(char* dest, uint8_t dest_size, uint32_t in);
```

Comment:
Prototype differs from definition :robot: 
I prefer the prototype btw :angel:

Reply:
Removed for snprintf :)